### PR TITLE
[ADD][business_requirement] Added To be reviewed field in BR

### PR DESCRIPTION
**Pull request by sudhir-serpentcs**
@victormartinelicocorp @elicoidal 
Added to be reviewed field with filter and grouping

### CODE REVIEW

**Comment by elicoidal** [comment]
@sudhir-serpentcs thanks for the update: requirement is well answered but I want to take the opportunity to redesign slightly the header of the BR form view as followed:
- [X] First line: description (unchanged)
- [X] Second line: Master Project / Parent Project
- [X] Third line: Customer / Reference
- [X] Fourth line: Category / Change Request
- [X] Fifth line: Requested by / Responsible
- [X] Sixth line: To be reviewed / Reviewers
- [X] Seventh line: Priority

Besides I want to change the following:
- [X] "Other information" => Logs
- [X] Hide "Level 1" information in "other information"
- [X] Add Responsible to the tree view (first)
- [ ] Add the total of Sales revenue at the bottom of the tree view

**Comment by sudhir-serpentcs** [comment]
@elicoidal  @victormartinelicocorp 
Improved form view as per requirement.
please review code.

**Comment by elicoidal** [comment]
@sudhir-serpentcs 
- [ ] Revenue in the tree view has disappeared and no total (obviously).
Rest of modifications are good!

Additional details:
- [X] created on => Creation Date
- [X] Created By and Creation Date should be on the same line (the 2 columns should be well aligned)
- [X] move linked project in the same line as priority (it will disappear later anyway)

**Comment by sudhir-serpentcs** [comment]
@elicoidal @victormartinelicocorp 
can you please review code.

**Comment by elicoidal** [comment]
Thanks @sudhir-serpentcs still missing the revenue and total in the tree view.

**Comment by elicoidal** [comment]
@sudhir-serpentcs there is very few left for this one. If you can fix it we could merge it quickly

**Comment by elicoidal** [comment]
> @sudhir-serpentcs there is very few left for this one. If you can fix it we could merge it quickly

Can you proceed so that we finish the roll-out?

**Comment by sudhir-serpentcs** [comment]
@elicoidal sure I will.

**Comment by elicoidal** [comment]
@sudhir-serpentcs I did not pay attention but some comments are duplicated in #105

**Comment by martintorresvm** [comment]
> @sudhir-serpentcs I did not pay attention but some comments are duplicated in #105

@sudhir-serpentcs Fix or modify `business_requirement_deliverable` only on #105

@elicoidal Once this PR only apply for the module `business_requirement` I think is ready to merge

**Comment by martintorresvm** [comment]
> @sudhir-serpentcs I did not pay attention but some comments are duplicated in #105 

@elicoidal Could you specify what is no need to be implemented here?

**Comment by serpentcs-dev1** [comment]
@elicoidal 

Yes the comments is covered in the PR #105 

Now I think all comments are fulfilled and it is ready to merge once reviewed.

cc @victormartinelicocorp

**Comment by elicoidal** [comment]
@victormartinelicocorp this one could be merge right?

**Review by elicoidal — APPROVED** [review]

File: business_requirement_deliverable/models/business.py
Comment (martintorresvm) [inline business_requirement_deliverable/models/business.py]: @sudhir-serpentcs This change is duplicated with https://github.com/OCA/business-requirement/pull/105/files#diff-005420dbf67a67f7fbc633822129feceL280

Please remove from this PR. The rest LGTM

cc @elicoidal

**Review by martintorresvm — CHANGES_REQUESTED** [review]

**Review by seb-elico — APPROVED** [review]

File: business_requirement_deliverable/models/business.py
Comment (elicoidal) [reply business_requirement_deliverable/models/business.py]: @sudhir-serpentcs once this is fixed, I will merge

**Comment by elicoidal** [comment]
@sudhir-serpentcs you will probably need to rebase given all modifications ;)

**Review by martintorresvm — APPROVED** [review]

**Comment by elicoidal** [comment]
@victormartinelicocorp Let's move on

**Pull request merged**
